customize.dist/translations/messages.pl.js

```diff
@@ -0,0 +1,229 @@
+define(function () {
+    var out = {};
+
+    // translations must set this key for their language to be available in
+    // the language dropdowns that are shown throughout Cryptpad's interface
+    out._languageName = 'Polish';
+
+    out.main_title = "Cryptpad: Wspólne edytowanie w czasie rzeczywistym, bez wiedzy specjalistycznej";
+    out.main_slogan = "Jedność siłą - Współpraca kluczem";
+
+    out.type = {};
+    out.type.pad = 'Pad';
+    out.type.code = 'Kod';
+    out.type.poll = 'Balot';
+    out.type.slide = 'Prezentacja';
+
+    out.errorBox_errorType_disconnected = 'Utracono połączenie';
+    out.errorBox_errorExplanation_disconnected = [
+        'Połączenie z serwerem zostało zerwane, możesz odświeżyć połączenie przeładowując stronę lub kontynuować przeglądanie dokumentu ',
+        'klikając poza obszar komunikatu.'
+    ].join('');
+
+    out.common_connectionLost = 'Przerwano połączenie z serwerem';
+
+    out.disconnected = 'Rozłączony';
+    out.synchronizing = 'Synchronizacja';
+    out.reconnecting = 'Wznawianie połączenia...';
+    out.lag = 'Lag';
+    out.readonly = 'Tylko do odczytu';
+    out.anonymous = "Anonimowy";
+    out.yourself = "Ty";
+    out.anonymousUsers = "użytkownicy anonimowi";
+    out.anonymousUser = "użytkownik anonimowy";
+    out.shareView = "URL tylko do odczytu";
+    out.shareEdit = "URL do edycji";
+    out.users = "Użytkownicy";
+    out.and = "i";
+    out.viewer = "czytający";
+    out.viewers = "osoby czytające";
+    out.editor = "edytujący";
+    out.editors = "osoby edytujące";
+
+    out.greenLight = "Wszystkie systemy działają poprawnie";
+    out.orangeLight = "Słabe łącze może wpłynąć na działanie aplikacji";
+    out.redLight = "Zostałeś rozłączony z sesją";
+
+    out.importButton = 'IMPORTUJ';
+    out.importButtonTitle = 'Importuj dokument z pliku lokalnego';
+
+    out.exportButton = 'EXPORTUJ';
```

Comment:
While it does seem difficult to read in uppercase, the text does not appear on any buttons on the website, only in tooltips. Browsing through the source code, it is only used as actual text in www/slide/oldmain.js (line 245), which seems unused.